client/src/assets/icons/pendingIcon.svg

```diff
@@ -0,0 +1,7 @@
+ <svg  xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
```

Comment:
Didnt you get an error with the second xmlns links?

Reply:
No, but i will check if it is necessary, thanks